src/frontend/src/lib/components/tokens/TokenCard.svelte

```diff
@@ -17,7 +19,12 @@
 		aria-label={`Open the list of ${token.symbol} transactions`}
```

Comment:
Unrelated: an i18n key is missing here

Reply:
it was fixed on the feature branch (PR #1863), so it will be fixed once merged